openwisp_notifications/templates/admin/openwisp_notifications/base_site.html

```
@@ -0,0 +1,90 @@
+{% block extrahead %}
+  {{ block.super }}
+  {% if request|should_load_notifications_widget and not media.js and 'jquery' not in block.super %}
+    <script src="{% static 'admin/js/vendor/jquery/jquery.min.js' %}"></script>
+    <script src="{% static 'admin/js/jquery.init.js' %}"></script>
+  {% endif %}
+{% endblock %}
+
+{% block user-tools %}
+  {{ block.super }}
+  {% if request|should_load_notifications_widget %}
+    <button class="ow-notifications toggle-btn" id="openwisp_notifications">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)